components/docs-chef-io/content/automate/ha_inplace_migration.md

```diff
@@ -184,6 +208,41 @@ Click [here](/automate/ha_backup_restore_object_storage/) to know more about the
 1. Reset the backup configuration path to Opensearch so that new backups will be stored in Opensearch directory, please click [here](/automate/ha_backup_restore_file_system/#configuration-for-automate-node-from-provision-host) to know more.
 {{< /note >}}
 
+## Validate successful migration
+
+1. Check the Automate UI of Automate HA. Check whether the data is present in Automate UI for HA.
+1. If you are using the embedded chef server, log in to the Chef Server HA node, and run the following script to get a count of objects from the Chef Infra Server, this should match the counts captured at the start of the migration
```

Comment:
Replace it with: 1. If you use the embedded chef server, log in to the Chef Server HA node, and run the following script to get a count of objects from the Chef Infra Server. This should match the counts captured at the start of the migration.